test/unit/core/reducers/data/cart.ts

```diff
@@ -0,0 +1,161 @@
+import { Actions, Store } from '../../../../../src/core';
+import Adapter from '../../../../../src/core/adapters/cart';
+import cart from '../../../../../src/core/reducers/data/cart';
+import suite from '../../../_suite';
+
+suite('cart', ({ expect, stub }) => {
+  let state: Store.Cart = {
+    content: {
+      cartId: '',
+      totalQuantity: 0,
+      items: [],
+      visitorId: '',
+      sessionId: '',
+      generatedTotalPrice: 0,
+      lastModified: null,
+    }
+  };
+
+  describe('updateCart()', () => {
+    it('should update cart state on GET_TRACKER_INFO', () => {
+      const payload = { visitorId: 'v', sessionId: 's' };
+      const newState = {
```

Comment:
You could just have made `newState = { ...state, visitorId: 'v', sessionId: 's' }`
No need to change it now, since you've written all of this, but in the future, just a heads up it can be simplified.